lib/BudgetCategories/budgetCategoryPrice.dart

```diff
@@ -1,5 +1,30 @@
-class BudgetCategoryPrice {
-  String? budgetName;
-  int? currentCategoryPrice;
-  int? totalCategoryPrice;
+import 'package:flutter/material.dart';
+
+
+String category = 'Food';
+String progressAmt = '249';
+String totalProgress = '500';
+
+// Calculates LinearPercentIndicator.percent
+double percentHelper(String dividend, String divisor) {
+  // Some text here
+  var intDividend = double.parse(dividend);
+  var intDivisor = double.parse(divisor);
+
+  return intDividend / intDivisor;
 }
+
+Color dynamicColor(String dividend, String divisor) {
```

Comment:
You could also set up the division outside of the function, that way it won't have to make a calculation every time it accesses the if statement.

Check out the way I did it in overallBudget.dart

Reply:
Sure thing, I'll make the update. The way I did it was an explicit change from string to double, thats why I left it like that